### DISCUSSION TITLE
Somes questions

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#67] [General]
I build my own organ with stops from different organs ..

I have one stop from Rieti (SP) with inégal temperament. 
the stop needs tuning but i think I have to change temperament. Is is possible on Goodf or I have to do this through GO only ? And can I do that for stops separately ?

Other question, if I want a same stop for example in the Pedal and in the Swell, but I don't want GO to load two different stops (folder with samples copied both in Pedal and Swell) how can I do ?

To explain, I want a Chamade (one stop) I can play on Swell OR pedal (two buttons on the console but one stop) ..

Many thanks

**Comment by larspalo** [comment]
> the stop needs tuning but i think I have to change temperament

If the stop has embedded pitch info (which is likely) then the only thing you need to adjust is for the "Original Temperament" which you do with the PitchTuning.

> same stop for example in the Pedal and in the Swell, but I don't want GO to load two different stops (folder with samples copied both in Pedal and Swell) how can I do ?

There are mainly two possible ways.
1. If you have separate ranks defined, you just add a new stop for the other division and reference to the **same** rank by using referenced ranks for the new stop too.
2. If you only have stops with internal ranks, you have to use the old style borrowing (with REF). Create a new stop for the other division and add the number of pipes needed - in the internal rank tab you then right click on the first pipe and select "Borrow pipe..." to bring up the borrowing dialog where you set the source and number of pipes to borrow.

With both methods you can only voice one of them (containing the actual original pipes). If you want separate voicing then you need to load the samples again as a new rank/stop.

**Reply by Jean-Gabriel1982** [reply]
Many thanks ! The solution was so easy I haven't thought about it. 
I was worried about windchests.. if I put the stop in Swell windchest how I can engage the same stop but just on pedal ..
It was a wrong problem, thanks again for the solution !
